skins/oasis/js/WikiHeader.js

```diff
@@ -298,62 +298,15 @@
 			},
 
 			firstMenuValidator: function () {
```

Comment:
You're going to like this :smiling_imp: 
How about extracting all of the preview logic to a separate file in `PageHeader` extension? `WikiHeader.js` will be removed during the clean up, anyway.